common/src/main/java/com/wynntils/features/user/tooltips/ItemStatInfoFeature.java

```diff
@@ -4,19 +4,49 @@
  */
 package com.wynntils.features.user.tooltips;
 
-import com.wynntils.core.components.Model;
 import com.wynntils.core.components.Models;
 import com.wynntils.core.config.Config;
 import com.wynntils.core.features.UserFeature;
 import com.wynntils.core.features.properties.FeatureCategory;
 import com.wynntils.core.features.properties.FeatureInfo;
 import com.wynntils.core.features.properties.FeatureInfo.Stability;
-import java.util.List;
+import com.wynntils.mc.event.ItemTooltipRenderEvent;
+import com.wynntils.utils.KeyboardUtils;
+import com.wynntils.utils.MathUtils;
+import com.wynntils.wynn.handleditems.WynnItemCache;
+import com.wynntils.wynn.handleditems.items.game.GearItem;
+import com.wynntils.wynn.utils.GearTooltipBuilder;
+import com.wynntils.wynn.utils.WynnItemUtils;
+import java.awt.Color;
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+import java.util.LinkedList;
+import java.util.Map;
+import java.util.NavigableMap;
+import java.util.Objects;
+import java.util.Optional;
+import java.util.TreeMap;
+import net.minecraft.ChatFormatting;
+import net.minecraft.network.chat.Component;
+import net.minecraft.network.chat.MutableComponent;
+import net.minecraft.network.chat.Style;
+import net.minecraft.network.chat.TextColor;
+import net.minecraftforge.eventbus.api.SubscribeEvent;
+import org.lwjgl.glfw.GLFW;
 
 @FeatureInfo(stability = Stability.STABLE, category = FeatureCategory.TOOLTIPS)
 public class ItemStatInfoFeature extends UserFeature {
     public static ItemStatInfoFeature INSTANCE;
 
+    private static final NavigableMap<Float, TextColor> COLOR_MAP = new TreeMap<>();
```

Comment:
There are 2 instances of this COLOR MAP. Do we need it in both of the places?

Reply:
No. It is my intention to rewrite this code further. The problem here is that the responsibility for generating the item lore is divided across different classes, which should have nothing to do with it. Instead of breaking abstraction barriers, I chose to copy the code for now.

Reply:
If you are not happy with this, I can open it up from one of the places and reference it from the other, but I'd rather not.

Reply:
It's fine for now then, you know the problem with it